### PR TITLE
OSOE-553: Apply automatic package.json dependency installation and update NE package reference

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-553](https://lombiq.atlassian.net/browse/OSOE-553)

[OSOE-553]: https://lombiq.atlassian.net/browse/OSOE-553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.BaseTheme/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
```
Comment (0liver) [inline Lombiq.BaseTheme/.eslintrc.js]: Why is this file here? It shouldn't have to be since we use a root _.eslintrc.js_ in OSOCE. Same for _.stylelintrc.js_.

**Review by 0liver — CHANGES_REQUESTED** [review]

File: Lombiq.BaseTheme/.eslintrc.js (hunk 1 above)
Comment (sarahelsaig) [reply Lombiq.BaseTheme/.eslintrc.js]: It was generated automatically. So it it shouldn't be here, then should I add it to the .gitignore?

Comment (0liver) [reply Lombiq.BaseTheme/.eslintrc.js]: @DAud-IcI I forgot that this question is still open.

Comment (sarahelsaig) [reply Lombiq.BaseTheme/.eslintrc.js]: but what is your answer?

Comment (DAud-IcI) [reply Lombiq.BaseTheme/.eslintrc.js:1]: @0liver ping

Comment (sarahelsaig) [reply Lombiq.BaseTheme/.eslintrc.js]: nvm, it's removed now

**Review by 0liver — APPROVED** [review]
